createWebNavigator.js

```diff
@@ -1,39 +1,38 @@
 import _ from 'underscore';
 import * as React from 'react';
-import {View, Pressable} from 'react-native';
+import {View, Dimensions} from 'react-native';
 import {useNavigationBuilder, createNavigatorFactory, StackRouter} from '@react-navigation/native';
+import RHPContainer from './components/RHPContainer';
+
+const isSmallScreenWidth = Dimensions.get('window').width <= 800;
+
 
 function CustomStackNavigator(props) {
     const {navigation, state, descriptors, NavigationContent} = useNavigationBuilder(StackRouter, {
       children: props.children,
       screenOptions: props.screenOptions,
       initialRouteName: props.initialRouteName,
     });
-
+    const isLHNVisible = (state.index === 0 && state.routes.length === 1) || !isSmallScreenWidth;
     const lastChatIndex = _.findLastIndex(state.routes, {name: 'Chat'});
     const lastRHPIndex = _.findLastIndex(state.routes, (route) => route.name !== 'Chat' && route.name !== 'LeftHandNav');
     const isRHPOnTopOfStack = lastRHPIndex === state.index;
     return (
         <NavigationContent>
             <View style={{flexDirection: 'row', flex: 1}}>
                 {state.routes.map((route, i) => {
-                    if (route.name === 'LeftHandNav') {
+                    if (isLHNVisible && i === 0) {
                         return (
                             <View
                                 key={route.key}
-                                style={{maxWidth: 350, flex: 1, borderRightWidth: 1}}
+                                style={{maxWidth: !isSmallScreenWidth ? 350 : Dimensions.get('window').width, flex: 1, borderRightWidth: !isSmallScreenWidth ? 1 : 0}}
```

Comment:
do we need to set maxWidth on small screens? I think we should set the maxWidth to `none` since it is `flex: 1` and the container is taking maxWidth already